packages/sw-cache-expiration/src/lib/behavior.js

```diff
@@ -14,144 +14,257 @@
 */
 
 import idb from 'idb';
-import Configuration from './configuration';
 import assert from '../../../../lib/assert';
-import {idbName, idbVersion} from './constants';
+import {idbName, idbVersion, urlProperty, timestampProperty} from './constants';
```

Comment:
Nit: Could you change these to urlPropertyName and timestampPropertyName?

I'm also finding it weird have normal variables lingering around and using like this rather than all caps variable names.